src/pages/play/Playground.tsx

```diff
@@ -0,0 +1,209 @@
+import useThemeContext from "@theme/hooks/useThemeContext";
+import clsx from "clsx";
+import React, { useCallback, useContext, useEffect, useRef, useState } from "react";
+import JSONTree from "react-json-tree";
+import MonacoEditor from "react-monaco-editor";
+import { version as tstlVersion } from "typescript-to-lua/package.json";
+import { version as tsVersion } from "typescript/package.json";
+import FengariWorker from "worker-loader?name=fengari.worker.js!./fengari.worker";
+import { debounce } from "../../utils";
+import { getInitialCode, updateCodeHistory } from "./code";
+import { monaco, useMonacoTheme } from "./monaco";
+import styles from "./styles.module.scss";
+
+// TODO:
```

Comment:
TODO or to remove?

Reply:
This is a new type imports syntax added in TS 3.8, but we can't use it right now because prettier doesn't support it yet